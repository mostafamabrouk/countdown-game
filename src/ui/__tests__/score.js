/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import { Score } from "../";
import "jest-styled-components";

test("Score renders withouut any parameters", () => {
  const component = renderer.create(<Score />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Score renders with minutes and seconds", () => {
  const component = renderer.create(<Score wins={0} loses={2} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
