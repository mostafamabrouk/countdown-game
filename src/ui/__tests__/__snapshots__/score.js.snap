// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score renders with minutes and seconds 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0;
}

.c0 li {
  margin: 10px;
}

<ul
  className="c0"
>
  <li>
    <strong>
      Win:
    </strong>
     
    0
  </li>
  <li>
    <strong>
      Loses:
    </strong>
     
    2
  </li>
</ul>
`;

exports[`Score renders withouut any parameters 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0;
}

.c0 li {
  margin: 10px;
}

<ul
  className="c0"
>
  <li>
    <strong>
      Win:
    </strong>
     
  </li>
  <li>
    <strong>
      Loses:
    </strong>
     
  </li>
</ul>
`;
